.astyleignore

```diff
@@ -11,8 +11,9 @@
 ^features/lwipstack/lwip
 ^features/lwipstack/lwip-sys
 ^features/mbedtls
-^features/nanostack/sal-stack-nanostack/source/Service_Libs/mdns/fnet
-^features/nanostack/targets
+^connectivity/nanostack/sal-stack-nanostack/source/Service_Libs/mdns/fnet
+^connectivity/nanostack/targets
```

Comment:
```suggestion
```
This doesn't exist anymore, it's probably a leftover from a previous PR.

Reply:
Done.